### PR TITLE
Construct url for local api

### PR DESCRIPTION
**Pull request by jeremybarbet**

### CODE REVIEW

**Comment by jeremybarbet** [comment]
@baldurh Should I add a local api example to see how it works? Maybe moving the swapi call from client to server, and adding a basic cache for results? What do you think?

File: shared/utils/localApiUrl.js [hunk 1]
```diff
@@ -3,10 +3,9 @@ import config from './config';
...
  */
-export default function getLocalApiUrl() {
-  return typeof window === 'undefined'
-    // Server
-    ? config('localApiUrl')
-    // Client
-    : '/api';
-}
+export default
+typeof window === 'undefined'
```
Comment (jeremybarbet) [inline shared/utils/localApiUrl.js]: Don't you have any lint issue with the indentation here?

**Comment by jeremybarbet** [comment]
LGTM 👍

File: shared/utils/localApiUrl.js (hunk 1 above)
Comment (baldurh) [reply shared/utils/localApiUrl.js]: No, but I got an error when I had two spaces

**Pull request merged**